modules/theme-tools/site-logo/inc/functions.php

```diff
@@ -103,7 +103,7 @@ function jetpack_has_site_logo() {
 function jetpack_the_site_logo() {
 	$logo    = site_logo()->logo;
 	$logo_id = get_theme_mod( 'custom_logo' ); // Check for WP 4.5 Site Logo
-	$logo_id = $logo_id ? $logo_id : $logo['id']; // Use WP Core logo if present, otherwise use Jetpack's.
+	$logo_id = $logo_id ? $logo_id : ( isset( $logo['id'] ) ? $logo['id'] : false ); // Use WP Core logo if present, otherwise use Jetpack's.
```

Comment:
Is this is `false` should we better handle the `$html` at line 123 instead of passing `false` to `wp_get_attachment_image`?

Reply:
The way I understand we never be passing `false` to `wp_get_attachment_image`.
Since it would be caught in ( ! jetpack_has_site_logo() ) check and we are not suing the `$logo_id` in that block. 
And keep $html as '';